### PR TITLE
feat: Add fetching of sports bets and add to cozy

### PR DESCRIPTION
**Pull request by LucsT**
@aenario review welcome.

Please note that : 
-> The add data has enough right? i suppose no, and should led to manifest changes.
-> Note the docType is 'com.unibet.bets' but we fetch unibet.fr, i think it's more future proof.
-> No docs for now, will do next

### CODE REVIEW

**Comment by aenario** [comment]
Remember to add the new doctype to the `cozy-store/locales`

**Comment by LucsT** [comment]
I will document the new doctype

File: src/index.js [hunk 1]
```diff
@@ -30,16 +33,24 @@ async function start(fields) {
...
   const entries = await parseDeposits()
+  log('info', 'Fetching sport bets ...')
+  // Horse bets and poker managed separatly.
+  const bets = await parseBets()
   log('info', 'Saving data to Cozy ...')
   await saveBills(entries, fields.folderPath, {
     identifiers: ['unibet'],
     contentType: 'application/pdf'
   })
+  await addData(bets, 'com.unibet.bets')
```
Comment (doubleface) [inline src/index.js]: addData blindly adds data in the doctype, even if it already exists. I think what you have to do here is also filter out already existing data. I have the same case with the deezer connector : https://github.com/konnectors/deezer/blob/e279ffcb9560e95650f227802bdf409ac8adba14/src/index.js#L42-L46

**Comment by LucsT** [comment]
@doubleface I now use hydrateAndFilter, are you ok with this version?

**Comment by LucsT** [comment]
DocType com.unibet.bets added to store here : https://github.com/cozy/cozy-store/pull/259

**Pull request merged**